modules/phase_field/test/tests/GBType/GB_Type_Phase1.i

```diff
@@ -0,0 +1,294 @@
+  [./Periodic]
+    [./top_bottom]
+      auto_direction = 'x y' # Makes problem periodic in the x and y directions
+    [../]
+  [../]
+[]
+
+[Materials]
+  [./CuGrGr]
+    # Material properties
+    type = GBEvolution # Quantitative material properties for copper grain growth.  Dimensions are nm and ns
+    GBmob0 = 2.5e-6 # Mobility prefactor for Cu from Schonfelder1997
+    GBenergy = 0.708 # GB energy for Cu from Schonfelder1997
+    Q = 0.23 # Activation energy for grain growth from Schonfelder 1997
+    T = 450 # Constant temperature of the simulation (for mobility calculation)
+    wGB = 6 # Width of the diffuse GB
+  [../]
+  [./GB_type]
+    # The new developed Miso Bnds Aux Kernel
+    type = ComputeGBMisorientationType
+    ebsd_reader = ebsd_reader
+    grain_tracker = grain_tracker
+    output_properties = 'gb_type'
+    outputs = exodus
+  [../]
+[]
+
+[Postprocessors]
+  # Scalar postprocessors
+  [./dt]
+    # Outputs the current time step
+    type = TimestepSize
+  [../]
+  [n_elements]
+    type = NumElems
+    execute_on = 'initial timestep_end'
+  []
+  [n_nodes]
+    type = NumNodes
+    execute_on = 'initial timestep_end'
+  []
+  [DOFs]
+    type = NumDOFs
+  []
+[]
+
+[Adaptivity]
+  initial_steps = 1
+  max_h_level = 1
+  marker = combined
+  [./Indicators]
+    [./error]
+      type = GradientJumpIndicator
+      variable = bnds
+    [../]
+  [../]
+  [./Markers]
+    [./bound_adapt]
+      type = ValueThresholdMarker
+      third_state = DO_NOTHING
+      coarsen = 0.999 #1.0
+      refine = 0.95 #0.95
+      variable = bnds
+      invert = true
+    [../]
+    [./errorfrac]
+      type = ErrorFractionMarker
+      coarsen = 0.1
+      indicator = error
+      refine = 0.7
+    [../]
+    [./combined]
+      type = ComboMarker
+      markers = 'bound_adapt errorfrac'
+    [../]
+  [../]
+[]
+
+[Executioner]
+  type = Transient # Type of executioner, here it is transient with an adaptive time step
+  scheme = bdf2 # Type of time integration (2nd order backward euler), defaults to 1st order backward euler
+
+  #Preconditioned JFNK (default)
+  solve_type = 'PJFNK'
+
+  petsc_options_iname = '-pc_type -pc_hypre_type -pc_hypre_boomeramg_strong_threshold'
+  petsc_options_value = '  hypre    boomeramg                   0.7'
+
+  l_max_its = 30 # Max number of linear iterations
+  l_tol = 1e-4 # Relative tolerance for linear solves
+  nl_max_its = 40 # Max number of nonlinear iterations
+  nl_abs_tol = 1e-11 # Relative tolerance for nonlienar solves
+  nl_rel_tol = 1e-10 # Absolute tolerance for nonlienar solves
+
+  [TimeStepper]
+    type = IterationAdaptiveDT
+    cutback_factor = 0.9
+    dt = 1
+    growth_factor = 1.1
+    optimal_iterations = 7
+  []
+
+  start_time = 0.0
+  num_steps = 2
+[]
+
+[Outputs]
+  perf_graph = true
+ [./console]
+    type = Console
+    max_rows = 10
+  [../]
+  [./exodus]
+     type = Exodus
+     execute_on = 'INITIAL TIMESTEP_END'
+  [../]
+  file_base = 'Phase1_output'
```

Comment:
any reason to use file base here? I find it rather confusing as it removes the clear relation between input and output file names.